deepchecks/vision/checks/data_integrity/property_label_correlation.py

```diff
@@ -0,0 +1,190 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module contains the property label correlation check."""
+
+from collections import defaultdict
+from typing import Any, Dict, Hashable, List, Optional, TypeVar
+
+import pandas as pd
+
+import deepchecks.ppscore as pps
+from deepchecks import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.core import DatasetKind
+from deepchecks.core.check_utils.feature_label_correlation_utils import get_pps_figure, pd_series_to_trace
+from deepchecks.core.errors import ModelValidationError
+from deepchecks.utils.strings import format_number
+from deepchecks.vision import Batch, Context, SingleDatasetCheck
+from deepchecks.vision.task_type import TaskType
+from deepchecks.vision.utils.image_properties import default_image_properties
+from deepchecks.vision.utils.property_label_correlation_utils import (calc_properties_for_property_label_correlation,
+                                                                      is_float_column)
+
+__all__ = ['PropertyLabelCorrelation']
+
+pps_url = 'https://docs.deepchecks.com/en/stable/checks_gallery/vision/' \
+          'train_test_validation/plot_feature_label_correlation_change.html'
+pps_html = f'<a href={pps_url} target="_blank">Predictive Power Score</a>'
+
+FLC = TypeVar('FLC', bound='PropertyLabelCorrelation')
```

Comment:
Change to PLC :)

Reply:
@Nadav-Barak it's FLC in PropertyLabelCorrelationChange as well

Reply:
@Nadav-Barak done